imod/mf6/uzf.py

```diff
@@ -341,16 +352,19 @@ def _check_options(
 
     def _create_uzf_numbers(self, landflag):
         """Create unique UZF ID's. Inactive cells equal 0"""
-        return np.cumsum(np.ravel(landflag)).reshape(landflag.shape) * landflag
+        active_nodes = xr.ones_like(landflag).where(landflag.notnull(), other =0.0)
```

Comment:
I think this can be shortened to:

```suggestion
        active_nodes = landflag.notnull().astype(np.int8)
```

Reply:
done